src/Text/Pandoc/Pretty.hs

```diff
@@ -492,6 +493,11 @@ braces = inside (char '{') (char '}')
 brackets :: Doc -> Doc
 brackets = inside (char '[') (char ']')
 
+-- | Apply 'brackets' to 'Doc' if boolean is true.
```

Comment:
I'd prefer not to add this to the Pretty module.  You only use it once, anyway, in the ConTeXt writer, and I think the code would be more intelligible without it.  (Or just define maybeBrackets in the writer where you use it.)